### PR TITLE
add apt tags

### PR DESCRIPTION
**Pull request by twaldrop**
This tag will allow us to limit a run to just adding new repos/keys

### CODE REVIEW

File: roles/repos/tasks/apt.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@
   register: result
   until: result|succeeded
   retries: 5
+  tags: apt
```
Comment (scoop206) [inline roles/repos/tasks/apt.yml]: In my experience, the more tags the better
Can we do the general 'apt' tag (like you've already done)
But add a specific one for each action.
So for this, the tags could be: `apt-keys`, `apt`

File: roles/repos/tasks/apt.yml [hunk 2]
```diff
@@ -24,6 +25,7 @@
...
```
Comment (scoop206) [inline roles/repos/tasks/apt.yml]: `apt-key-packages`, `apt`

File: roles/repos/tasks/apt.yml [hunk 3]
```diff
@@ -36,3 +38,4 @@
...
```
Comment (scoop206) [inline roles/repos/tasks/apt.yml]: `apt-repos`, `apt`

**Comment by twaldrop** [comment]
retest

**Pull request merged**